src/js/letters/utils/helpers.js

```diff
@@ -46,3 +47,22 @@ export const letterContent = {
   benefit_summary: 'This letter shows what benefits you\'re receiving from the VA, military service, and disability status. Below, you can choose if you want military service and disability status to be included.',
   benefit_verification: 'This letter shows what benefits you\'re receiving from the VA. It is different from the benefit summary because it includes [x] and does not give you the option to choose what is included in the letter.'
 };
+
+export function veteranBenefitSummaryOptionText(allOptions, currentOption) {
```

Comment:
Yep, good question, I was wondering about that too. The reason why I thought I had to pass `allOptions` in is because some of the text embeds data from the options. For example, for `awardEffectiveDate`, the actual date is embedded in the text. I'm not sure how else to handle it, as these aren't even really strings of text but rather React markup, so I don't think I could push it on the end or something like that.